lib/sles4sap_publiccloud_basetest.pm

```diff
@@ -56,6 +95,30 @@ sub import_context {
     );
 }
 
+=head2 set_cli_ssh_opts
+
+    $self->set_clt_ssh_opts
+
+Set default command line SSH options in the instance stored in C<$self-E<gt>{my_instance}>. It
+also requires a provider in C<$self-E<gt>{provider}> to access the SSH key file path used in
+the deployment.
+
+=cut
+
+sub set_cli_ssh_opts {
+    my ($self) = @_;
+    croak("Expected \$self->{my_instance} is not defined. Check module Description for details")
+      unless $self->{my_instance};
+    croak("Expected \$self->{my_provider} is not defined. Check module Description for details")
+      unless $self->{my_provider};
+    my $ssh_opts = join(' ',
+        '-o', 'UserKnownHostsFile=/dev/null',
+        '-o', 'StrictHostKeyChecking=no',
```

Comment:
You're hiding potential issues with those lines.

Reply:
Can you elaborate? Main purpose of this PR is to get us back into a working condition in these tests, but I'd like to follow up on this until we're back in green.

Reply:
Removed.